lib/src/components/NxCollapsibleItemsSelect/NxCollapsibleRadioSelect/__tests__/NxCollapsibleRadioSelect.test.tsx

```diff
@@ -4,173 +4,495 @@
+      expect(renderAndGetTrigger()).toHaveAttribute('aria-expanded', 'false');
+      expect(renderAndGetTrigger({ isOpen: false })).toHaveAttribute('aria-expanded', 'false');
+      expect(renderAndGetTrigger({ isOpen: true })).toHaveAttribute('aria-expanded', 'true');
+    });
+
+    it('sets disabled if the disabled prop is set to true or options is an empty array', function() {
+      expect(renderAndGetTrigger()).toBeEnabled();
+      expect(renderAndGetTrigger({ disabled: false })).toBeEnabled();
+      expect(renderAndGetTrigger({ disabled: null })).toBeEnabled();
+      expect(renderAndGetTrigger({ disabled: true })).toBeDisabled();
+
+      expect(renderAndGetTrigger({ options: [] })).toBeDisabled();
+      expect(renderAndGetTrigger({ options: [], disabled: true })).toBeDisabled();
+      expect(renderAndGetTrigger({ options: [], disabled: false })).toBeDisabled();
+      expect(renderAndGetTrigger({ options: [], disabled: null })).toBeDisabled();
+    });
+
+    it('fires the component\'s onToggleCollapse when clicked if there are options', async function() {
+      const user = userEvent.setup(),
+          onToggleCollapse = jest.fn(),
+          noOptionsTrigger = renderAndGetTrigger({ onToggleCollapse, options: [] }),
+          optionsTrigger = renderAndGetTrigger({ onToggleCollapse });
+
+      expect(onToggleCollapse).not.toHaveBeenCalled();
+
+      await user.click(noOptionsTrigger);
+      expect(onToggleCollapse).not.toHaveBeenCalled();
+
+      await user.click(optionsTrigger);
+      expect(onToggleCollapse).toHaveBeenCalled();
+    });
+
+    it('does not fire the component\'s onToggleCollapse when clicked and the element is disabled', async function() {
+      const user = userEvent.setup(),
+          onToggleCollapse = jest.fn(),
+          trigger = renderAndGetTrigger({ onToggleCollapse, disabled: true });
 
-      expect(optionWrapper).toHaveDisplayName('ForwardRef(NxRadio)');
-      expect(optionWrapper).toHaveProp('isChecked', false);
+      expect(onToggleCollapse).not.toHaveBeenCalled();
+
+      await user.click(trigger);
+      expect(onToggleCollapse).not.toHaveBeenCalled();
     });
 
-    it('renders checked <NxRadio> if option is selected', function () {
-      const shallowRender = getShallow({selectedId: 'foo'});
-      const optionWrapper = shallowRender.renderProp('renderOption')({id: 'foo', name: 'Foo'});
+    it('renders children prop into trigger', function() {
+      const trigger = renderAndGetTrigger(),
+          triggerContent = screen.getByTestId('trigger-content');
+
+      expect(trigger).toContainElement(triggerContent);
+    });
 
-      expect(optionWrapper).toHaveDisplayName('ForwardRef(NxRadio)');
-      expect(optionWrapper).toHaveProp('isChecked', true);
+    describe('disabledTooltip', function() {
+      // the user API does not trigger mouseenter / mouseover events on disabled elements, so need to hover on
```

Comment:
Cindy and I discussed this together, and I saw your comment on the RTL conversion for `NxCollapsibleMultiSelect` here (https://github.com/sonatype/sonatype-react-shared-components/pull/929#discussion_r1087853962), so changed the test to hover on the wrapper div and added a comment